Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -669,7 +669,7 @@ WarpXParticleContainer::DepositCharge ( WarpXParIter& pti, RealVector& wp,
       const std::array<Real,3>& cxyzmin_tile = WarpX::LowerCorner(ctilebox, lev-1);
 
 #ifdef AMREX_USE_GPU
-      data_ptr = (*crhomf)[pti].dataPtr();
+      data_ptr = (*crhomf)[pti].dataPtr(icomp);
```

Comment:
Thanks for fixing this bug!

Reply:
Not having icomp in the parenthesis does not affect the simulation the way it is set up now. But, probably 'crhomf' is to be accessed for a certain input condition, the error may show up if (icomp) is not included.